acceptance/status_server_test.go

```diff
@@ -31,30 +31,70 @@ import (
 	"github.com/cockroachdb/cockroach/acceptance/localcluster"
 	"github.com/cockroachdb/cockroach/proto"
 	"github.com/cockroachdb/cockroach/util"
-	"github.com/cockroachdb/cockroach/util/log"
 )
 
 type details struct {
 	NodeID proto.NodeID `json:"nodeID"`
 }
 
+// getNewClient returns a new http client for fetching data from the status
+// server. This client should be reused.
+func getNewClient() *http.Client {
+	return &http.Client{
+		Timeout: 200 * time.Millisecond,
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
+			},
+		},
+	}
+}
+
 // get performs an HTTPS GET to the specified path for a specific node.
 func get(t *testing.T, client *http.Client, node *localcluster.Container, path string) []byte {
 	url := fmt.Sprintf("https://%s%s", node.Addr(""), path)
-	resp, err := client.Get(url)
-	if err != nil {
-		t.Fatalf("could not GET %s - %s", url, err)
-	}
-	defer resp.Body.Close()
-	if resp.StatusCode != http.StatusOK {
-		t.Fatalf("could not GET %s - statuscode: %d", url, resp.StatusCode)
-	}
-	body, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		t.Fatalf("could not open body for %s - %s", url, err)
+	// There seems to be some issues while trying to connect to the status
+	// server, so retry (up to 5 times) with a 1 second delay each time.
+	// TODO(Bram): Clean this up once we get to the bottom of the issue.
+	for retry := 0; retry < 5; retry++ {
+		resp, err := client.Get(url)
+		if err != nil {
+			if retry < 4 {
+				t.Logf("could not GET %s - %s", url, err)
+				time.Sleep(100 * time.Millisecond)
+				client = getNewClient()
```

Comment:
There was an error with use of a closed connection.  I wanted to see if a new client would fix the issue.